src/main/java/com/mercadolibre/grupo1/projetointegrador/dtos/InboundOrderDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.mercadolibre.grupo1.projetointegrador.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+/**
+ * @author Nayara Coca
+ * Classe responsável por filtrar dados dos pedido
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class InboundOrderDTO {
+    //verificar se será gerado automaticamente
+    private Integer orderNumber;
+    @NotNull
+    private LocalDate orderDate;
+    @NotNull
+    private String section;
+}
```

Comment:
Faltou só o atributo batchStock que é um Set de BatchStockDTO!
```suggestion
    @Valid //Essa anotacao diz ao spring para validar os atributos do objeto interno BatchStock
    @NotEmpty
    private Set<BatchStockDTO> batchStock;
}
```